packages/Table/src/index.tsx

```diff
@@ -117,10 +118,15 @@ const Table: FunctionComponent<ITableProps> = ({
     </div>
   );
 
-  const tableWrapperClassName = `table__wrapper ${
-    stickyScroll ? "table__wrapper--scroll" : ""
-  }`.trim();
-  const tableClassName = `table ${stickyScroll ? "table--scroll" : ""}`.trim();
+  const tableWrapperClassName = classnames({
+    table__wrapper: !stickyScroll,
```

Comment:
shouldn't `table__wrapper` be a string?

Reply:
It is automatically changed from string to just "table__wrapper" when I save the file :shrug:

Reply:
😮 
good to know